packages/lab/src/list-next/ListNext.tsx

```diff
@@ -0,0 +1,120 @@
+import { makePrefixer, useForkRef, useId } from "@salt-ds/core";
+import {
+  FocusEvent,
+  forwardRef,
+  HTMLAttributes,
+  KeyboardEvent,
+  MouseEvent,
+  useRef,
+} from "react";
+import { clsx } from "clsx";
+import { useList } from "./useList";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import listNextCss from "./ListNext.css";
+import { ListNextContext } from "./ListNextContext";
+
+const withBaseName = makePrefixer("saltListNext");
+
+export interface ListNextProps extends HTMLAttributes<HTMLUListElement> {
+  /**
+   * If true, all items in list will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * Optional id prop
+   *
+   *The id for the list component
+   */
+  id?: string;
+  /* Value for the uncontrolled version. */
+  selected?: string;
+  /* Initial value for the uncontrolled version. */
+  defaultSelected?: string;
+}
+
+export const ListNext = forwardRef<HTMLUListElement, ListNextProps>(
+  function ListNext(
+    {
+      children,
+      className,
+      disabled: listDisabled,
+      id,
+      onSelect,
+      onFocus,
+      onBlur,
+      onKeyDown,
+      onMouseDown,
+      selected: selectedProp,
+      defaultSelected,
+      ...rest
+    },
+    ref
+  ) {
+    const targetWindow = useWindow();
+    useComponentCssInjection({
+      testId: "salt-list-next",
+      css: listNextCss,
+      window: targetWindow,
+    });
+
+    const listId = useId(id);
+    const listRef = useRef<HTMLUListElement>(null);
+    const handleRef = useForkRef(listRef, ref);
+
+    const {
+      handleFocus,
+      handleKeyDown,
+      handleBlur,
+      handleMouseDown,
+      activeDescendant,
+      contextValue,
+    } = useList({
+      disabled: listDisabled,
+      selected: selectedProp,
+      defaultSelected,
+      id: listId,
+      ref: listRef,
+    });
+
+    const focusHandler = (event: FocusEvent<HTMLUListElement>) => {
```

Comment:
We normally use `handleFocus`. It appears you've switched the words. Was it intentional?

Reply:
because handleFocus has already been declared in the useListHook.

Reply:
Maybe rename the other one as `handleFocusProp`? Would be more inline with the current pattern

Reply:
@ZarkoPernar